impeller/compiler/shader_lib/impeller/transform.glsl

```diff
@@ -26,4 +26,32 @@ vec4 IPPositionForGlyphPosition(mat4 mvp,
               unit_position.y * destination_size.y, 0.0, 1.0);
 }
 
-#endif
+#ifdef IMPELLER_TARGET_OPENGLES
```

Comment:
Its not possible to leave a define unevaluated with our shader compiler pipeline. that is, any define must evaluate to a value during the engine build - so its only possible to use these to turn this on/off for an entire backend.

If there is a runtime capabilities check you'd need to use, you'll have to use a specialization constant - which must be defined in the shader entrypoint and provided during initialization in content_context.cc

For an example of  that, grep for supports_decal.

In this particular case, that might not be worthwhile.

Reply:
Thank you Jonah. I'm in a lot of meetings this week but will try to update this when possible.

Reply:
I don't think adding a second pipeline is worth the reduced flops of the O(1) invocation of this in conical gradient for on GLES only.

`inverse` isn't supported until GLSL ES 3. This restores GLES 2 support (GLSL ES 1).

Reply:
Yes I agree, one ifdef for GLES